### DISCUSSION TITLE
Allow access to account repositories

### DISCUSSION THREAD

**Discussion by SamHobbsOrg** [community/community#21985] [New to GitHub]
<p>I apologize for not knowing the correct terminology. If I knew the correct terminology then it is nearly certain I could find the answer myself.</p>
<p>I have set my GitHub account to <strong>selectively</strong> allow access; in other words, I select what repositories that another user has access to. I cannot find where to <strong>specify</strong> what repositories another user has access to. I intend to stop using that feature and allow access to all repositories but I cannot find where to do that.</p>
<p>I will be specific about the problem I am trying to solve. I am attempting to use the DigitalOcean CLI to create an applicaiton. I am getting the error:</p>
<pre><code class="lang-auto">Error: POST https://api.digitalocean.com/v2/apps: 400 (request "b6e3ce89-a275-4a46-87f1-7a9be1094800") GitHub user does not have access to SamHobbsOrg/BlazorWasmDigitalOcean
</code></pre>
<p>My BlazorWasmDigitalOcean is a public repository so that is not the problem. I assume that I can solve the problem by removing the feature that requires me to specify what repositories in my account can be accessed. Where do I do that, or at least where is the documentation of how to do that?</p>

**Comment by SamHobbsOrg** [comment, marked as answer]
<p><strong>I found it.</strong></p>
<p>The problem is that the error message says <em>user</em> but it is the DigitalOcean <strong>app</strong> that needs access. I must have looked everywhere else, just not apps. When I looked there the problem is easy to solve.</p>

**Reply by Kyserbyte** [reply]
I have exactly the same problem and I can't fix it.
What was your specific solution for this?

**Reply by King-Mob** [reply]
For the next person: it was to go here https://cloud.digitalocean.com/apps/github/install and add the app to the repo you want it to connect to

**Comment by queenofcorgis** [comment]
<p><a class="mention" href="/samhobbsorg">@SamHobbsOrg</a> nice self-solve :star_struck: Thanks for letting us know the solution and marking it as such :+1:</p>
